capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -2,60 +2,60 @@
+| Relationship Discovery                     | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Instance Drift Comparison                  | Yes                       | Yes             | No                          | Yes                        | No                                    | No                                |
+| Track Instance Genealogy                   | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | No                                |
+| Capacity & Utilization                     | Yes                       | Yes             | Yes                         | Yes                        | No                                    | No                                |
+| Capture Cloud Layer Timelines              | No                        | No              | No                          | Yes                        | Yes                                   | No                                |
+| Capture Instance Event Timelines           | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | No                                |
+| Discovery - Provider                       | No                        | No              | No                          | No                         | No                                    | No                                |
+| Disk Addition to Instance                  | Yes                       | Yes             | No                          | Yes                        | No                                    | Yes                               |
+| Key Pairs Inventory                        | Yes                       | No              | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Key Pairs Management                       | Yes                       | No              | No                          | Yes                        | Yes                                   | Yes                               |
+| Reporting                                  | Yes                       | Yes             | Yes                         | Yes                        | No                                    | No                                |
+| Right Sizing                               | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | No                                |
+| Chargeback                                 | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
```

Comment:
@gtanzillo Do you know if this Chargeback row is describing C&U + Allocation chargeback? Since the Power Virtual Servers provider will be capable of allocation based chargeback reports I marked it `Yes`, but that I don't think that fits of the "IBM PowerVC via the OpenStack API" column being marked `No`.

Reply:
@jaywcarman on the InfraManager cap matrix SCVMM (which also doesn't support C&U) says "Yes (but limited to inventory allocation)" [[ref]](https://github.com/ManageIQ/manageiq-documentation/blob/master/capabilities_matrix/_topics/infrastructure_providers.md) so maybe we could use that

Reply:
:+1: That looks good to me - done.